### PR TITLE
Feature | PEO-1018

### PR DESCRIPTION
**Pull request by Bulatkin**
JIRA ticket - https://garage.atlassian.net/browse/PEO-1018

### Before submitting the merge request make sure the following are checked
- [ ] Followed the style guidelines of this project
- [ ] Performed a self-review of own code
- [ ] Wrote the tests that prove that fix is effective/that feature works
- [ ] Check correctness of commit names/branch name/assignees list/labels
### Notable changes
- Added PullRequest creation tests.
- Added ```RSpec/MessageSpies EnforcedStyle: receive``` and ``` RSpec/MultipleExpectations Enabled: false``` to Rubocop config
### Commands to run

### CODE REVIEW

File: spec/rubyhub/pull_request_create.rb [hunk 1]
```diff
@@ -1,16 +0,0 @@
-  RSpec.describe Rubyhub::Operations::PullRequest::Create do
-    let(:options) { { template: 'your_template_name' } }
-
-    describe '.call' do
-      context 'when raises error' do
-        it 'file doesnt exist' do
-          expect { described_class.call(options) }.to raise_error(ConfigFileDoesNotExistError)
```
Comment (Volosh1n) [inline spec/rubyhub/pull_request_create.rb:7]: the only reason to remove specs -- when we remove whole class, which was covered with this exact spec... But Rubyhub::Operations::PullRequest::Create wasn't removed. Please, move this file back

Comment (Bulatkin) [reply spec/rubyhub/pull_request_create.rb:7]: The test itself wasn't removed completely. According to suggestion of Rubocop it was moved to the appropriate folder ```spec/rubyhub/operations/pull_request/create_spec.rb```

**Review by Volosh1n — APPROVED** [review]
Please, merge this ONLY after #12  fixed. Because this PR contains same config for CI, which have nod good approaches

**Comment by Bulatkin** [comment]
Moved changes to pull request Feature | PEO-1023

**Pull request closed without merging**